cmd/gentypes/gentypes.go

```diff
@@ -293,6 +293,11 @@ func emitToplevelType(typeName string, descJson json.RawMessage) string {
 				// the specification) assume a value of 'true' for backward
 				// compatibility. See: https://github.com/google/go-dap/issues/39
 				jsonTag += "\"`"
+			} else if typeName == "InitializeRequestArguments" && (propName == "linesStartAt1" || propName == "columnsStartAt1") {
```

Comment:
what about pathFormat?

Reply:
I did not include pathFormat because when left empty ("") then the default value of "path" will be added. The other two values are booleans so the empty value (false), will never be sent. And since we default to true, then they will always resolve to be true.

This is my understanding but I could be wrong.

Reply:
No, you are right. Empty and missing path are both empty in Go and either way are treated the same mapping to "path". But empty (false) and missing bool are not to be treated the same. Makes sense.